tic-tac-toe/src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+import style from './styles.scss';
+
+function LoginForm(props) {
+  // eslint-disable-next-line
```

Comment:
what happend here? jaja okey, we try to never use eslint-disable. There are a few exceptions but those are really rare. If you are going to use eslint-disable you should be specific: you have to search for the eslint error and disable just for that error. In this particular case, what's the error?